pkg/controllers/drift/controller.go

```diff
@@ -39,74 +38,94 @@ func NewController(kubeClient client.Client, cloudProvider *cloudprovider.CloudP
 
 func (d Drift) Reconcile(ctx context.Context, req controllerruntime.Request) (reconcile.Result, error) {
 	ctx = logging.WithLogger(ctx, logging.FromContext(ctx).Named(controllerName).With("provisioner", req.Name))
-
-	provisioner := &v1alpha5.Provisioner{}
-	if err := d.kubeClient.Get(ctx, req.NamespacedName, provisioner); err != nil {
+	node := &v1.Node{}
+	if err := d.kubeClient.Get(ctx, req.NamespacedName, node); err != nil {
```

Comment:
Don't we want to retry on error here?

Reply:
Yeah I was going to say the same. We shouldn't be masking any errors without a good reason.

Reply:
Sorry missed those errors.

Reply:
> Don't we want to retry on error here?

Should I check for a not found error, if not then requeue ?